core_data_modules/cleaners/codes/kenya_codes.py

```diff
@@ -295,7 +296,7 @@ class KenyaCodes(object):
     SEME = "seme"
     SHINYALU = "shinyalu"
     SIGOR = "sigor"
-    SIGOWET_SION = "sigowet-sion"
+    SIGOWET_SION = "sigowet sion"
```

Comment:
Actually hyphens are fine in codes, provided this would normally be written with a hyphen.

Reply:
Okay, It can be written with/without hyphen. Lets keep have it without an hyphen